app/react/Markdown/MarkdownViewer.js

```diff
@@ -38,6 +41,20 @@ class MarkdownViewer extends Component {
     return <Element {...props} key={index} />;
   }
 
+  componentDidMount() {
+    if (errorCollector.hasErrors()) {
+      errorCollector.report();
```

Comment:
I couldn't find a util function to log a message as Info that could be used in the client. How about reporting the error as console.info to avoid executing the noop or removing the function?